webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/RegistrationCommandsPageConstants.js

```diff
@@ -0,0 +1,8 @@
+import URI from 'urijs';
+import { foremanUrl } from '../../../../foreman_tools';
+
+export const docUrl = () => {
+  const rootUrl = "https://docs.theforeman.org/nightly/Managing_Hosts/index-foreman-el.html#registering-a-host_managing-hosts"
+
+  return(foremanUrl(`/links/manual?root_url=${URI.encode(rootUrl)}`))
```

Comment:
If we're going to abuse the manual type in the links controller to do this, lets at least keep it's api - i.e. split between `root_url` and `section` rather than passing the whole url as the `root_url`.